OpenTerm/Data/UserDefaultsController.swift

```diff
@@ -68,6 +68,21 @@ class UserDefaultsController {
 			userDefaults.synchronize()
 		}
 	}
+	
+	var caretStyle: Int {
```

Comment:
It would be better to make the type of this property `CaretStyle`.

Reply:
I tried that at first, but i haven't managed to get a tidy implementation. this is the tidiest I could get.
Setting the userDefault to CaretStyle returns an error because you can't set a user default of CaretStyle type, so I just gave CaretStyle a Int type and used int. If i set the user default to CaretStyle I'd have to insert lots of "CaretStyle(rawvalue: value)" and it isn't prettier imo.

Any Thoughts?